src/ffi/pyhash.rs

```diff
@@ -1,6 +1,18 @@
 use crate::ffi::pyport::{Py_hash_t, Py_ssize_t};
-use std::os::raw::{c_char, c_int, c_void};
+use std::os::raw::{c_char, c_int, c_ulong, c_void};
 
+// skipped non-limited _Py_HashDouble
+// skipped non-limited _Py_HashPointer
+// skipped non-limited _Py_HashPointerRaw
+// skipped non-limited _Py_HashBytes
```

Comment:
Looks like we do define this down below, perhaps move it up here?

Reply:
Yeah, I missed this!